sdk/tests/deqp/functional/gles3/es3fNegativeTextureApiTests.js

```diff
@@ -1719,28 +1719,27 @@ goog.scope(function() {
 
         testGroup.addChild(new es3fApiCase.ApiCaseCallback('texparameterf', 'Invalid gl.texParameterf() usage', gl,
         function() {
-
-            /** @type{ WebGLTexture} */ var texture;
-            texture = gl.createTexture();
-            gl.bindTexture(gl.TEXTURE_2D, texture);
```

Comment:
Could you please put this code back? This should allow all of the changes to this test to be undone. Thanks.